### PR TITLE
Upgrade Route53 to v0.5.0

### PR DESCRIPTION
**Pull request by janeczku**

### CODE REVIEW

File: templates/route53/7/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## Route53 DNS
+
+Rancher External DNS service powered by Amazon Route53
+
+#### IAM policy
+
+This IAM policy describes the minimum permissions required for this service. Replace `<HOSTED_ZONE_ID>` with the ID of the hosted zone or use a wildcard ('*').
```
Comment (alena1108) [inline templates/route53/7/README.md]: Can we get an example of how IAM policy can be specified in the template?

**Comment by alena1108** [comment]
LGTM

File: templates/route53/7/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      description: "The resource record cache time to live (TTL), in seconds"
+      type: "int"
+      default: 60
+      required: false
+    - variable: "ROOT_DOMAIN"
+      label: "Hosted Zone Name"
+      description: "Route53 hosted zone name (zone has to be pre-created). DNS entries will be created for <service>.<stack>.<environment>.<hosted zone>"
+      type: "string"
+      required: true
+    - variable: "ROUTE53_ZONE_ID"
```
Comment (vincent99) [inline templates/route53/7/rancher-compose.yml:34]: A little late now that it's baked in to the image, but this naming is odd.. Other fields are not prefixed ROUTE53, and the hosted zone is called ROOT_DOMAIN above.  `ROOT_DOMAIN_ID`?

Comment (janeczku) [inline templates/route53/7/rancher-compose.yml:34]: @vincent99 `ROOT_DOMAIN` is a global option (ie. not provider specific). It's used to construct the FQDN: `<service>.<stack>.<environment>.<root-domain>`.

**Pull request merged**